src/rules.js

```diff
@@ -277,10 +277,17 @@ var rules = {
     return !!val.match(req);
   },
 
-  date: function(val) {
-    var valid = (new Date(val).toString()) !== 'Invalid Date';
-    if (typeof val === 'number') {
-      return val.toString().length === 12 && valid;
+  date: function(val, format) {
+    var moment;
+    var valid = false;
+    try {
+      moment = require('moment');
```

Comment:
hey so will this work in the browser, or will it throw an error like "undefined (require) is not a function"?